src/python/pants/backend/python/target_types.py

```diff
@@ -1268,6 +1268,14 @@ class PythonRequirementResolveField(PythonResolveField):
     )
 
 
+class PythonRequirementFindLinksField(StringSequenceField):
+    # NB: This is solely used for `pants_requirements` target generation
```

Comment:
This is scope creep... but would it be reasonable for someone to set `find_links` on a  `python_requirement` or `python_requirements` target themselves? 

As in, this could be publicly exposed (maybe as follow-up, to avoid blocking this), as a more scoped version of https://www.pantsbuild.org/docs/reference-python-repos ?

Reply:
We could, but for now I'm taking the cautious approach of making this private.